src/js/trove/table.js

```diff
@@ -596,11 +596,39 @@
         
         'get-column': runtime.makeMethod1(function(_, col_name) {
           ffi.checkArity(2, arguments, "get-column", true);
+          runtime.checkString(col_name);
           if(!hasColumn(col_name)) {
             ffi.throwMessageException("The table does not have a column named `"+col_name+"`.");
           }
           return runtime.ffi.makeList(getColumn(col_name));
         }),
+
+        'select-columns': runtime.makeMethod1(function(_, colnames) {
```

Comment:
Why does `select-columns` consume a `List` of column names, whereas `row` consumes a variadic number of values?